### ISSUE TITLE
56.2 doesn't start - Linux Mint 18.2, KDE

### ISSUE THREAD

**Issue by atomGit** [BrowserWorks/waterfox#570]
upgraded to 56.2 via package manager and WF won't run - if i start it via shell...

```
$ waterfox
XPCOMGlueLoad error for file /opt/waterfox/liblgpllibs.so:
/lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by /opt/waterfox/liblgpllibs.so)
Couldn't load XPCOM.
```

waterfox-kde 56.2

libc6 2.23-0ubuntu10 is installed

possibly related: #565 , #223

**Comment by hawkeye116477** [comment]
@atomGit Fixed with 56.2.0.1 (used magic LaunchPad VMs :smile:)

**Comment by atomGit** [comment]
indeed it is - thanks!
